engines/router/missionctl/handlers/http_handler.go

```diff
@@ -177,7 +177,9 @@ func (h *httpHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 	payload := resp.Body()
 
 	// Write the json response to the writer
-	rw.Header().Set("Content-Type", resp.Header().Get("Content-Type"))
+	for key := range resp.Header() {
+		rw.Header().Set(key, resp.Header().Get(key))
```

Comment:
I think we can use [Header.Clone](https://pkg.go.dev/net/http#Header.Clone) for this instead.

Reply:
`Header.Clone` returns a copy of the existing header, but there is no access to the `ResponseWriter`'s header instance, where this cloned header can be assigned to. Hence, we can only use `http.Header` methods to set it one by one.

Reply:
Oh I didn't realise thanks. Thanks for clarifying.